Bucketeer/Sources/Internal/Model/Evaluation.swift

```diff
@@ -6,9 +6,9 @@ struct Evaluation: Hashable, Codable {
     let featureVersion: Int
     let userId: String
     let variationId: String
-    var variation: Variation
+    let variationName: String
+    let variationValue: String
     let reason: Reason
-    var variationValue: String
```

Comment:
Because of unit tests, this var was mutable. I changed the tests so we can make it immutable.